### DISCUSSION TITLE
no way to have persistent tab groups that will never get closed 😔

### DISCUSSION THREAD

**Discussion by jjt** [mbnuqw/sidebery#1921] [Q&A]
i have a number of persistent tab groups that i never ever want to close, regardless of whether i hit "close tabs" or "close tabs below" from a tab above them

i do not want to pin them, because pinning ungroups them

i see "Warn on trying to close multiple tabs: collapsed", which works if i happen to have any groups closed  but it does not work if they are all open, or i hit "close tabs below" on a tab above open groups

i do not want to get warned every time i close multiple tabs, because i close multiple tabs frequently

i do not want to rely on noticing this has happened during the period of time that i can undo the closure

**Comment by sector101010** [comment]
Is this what panels are meant for? It would be nice if panels could be displayed as a list, instead of tab groups.
